pkg/inspecttypes/native/image.go

```diff
@@ -0,0 +1,29 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package native
+
+import (
+	"github.com/containerd/containerd/images"
+)
+
+// Image corresponds to a containerd-native image object.
+// Not compatible with `docker image inspect`.
+type Image struct {
+	images.Image
+	ImageSpec  interface{} `json:"Image"`
```

Comment:
Why `interface{}`?

Reply:
also `json:"ImageSpec"`

Reply:
@AkihiroSuda  `interface{}`. because i want to avoid using the `ocispec.Image` in this native containerd object

Reply:
Why do you want to avoid it?

Reply:
it looks cleaner if i use an interface here and map this interface to `ocispec.Image` in dockercompat may be wrong. Anyway I made the change